### DISCUSSION TITLE
Shortcode Field Variable to Pull Relationship (Simple Custom Defined List) Value

### DISCUSSION THREAD

**Discussion by picthall** [pods-framework/pods#6801] [Ideas]
## Problem

Refer to Slack thread: https://app.slack.com/client/T02SPP4MV/C02SVLHQF/thread/C02SVLHQF-1619201186.005800

I extended the Post type with a custom field, specifically a Relationship field that uses a simple custom defined list. For purposes of example, the name of the field is Location and one of the custom defined options is "middle-east|Middle East". If I use [pods field="location"] it pulls the label, or in my example "Middle East". There is no equivalent shortcode that pulls the value, or in my example, "middle-east". Please add the ability to do this.

## Proposed Solution

(see above)

## Workaround

From the Slack thread (referring to my example), two workarounds to get this value are:
[pods field="_pods_location"]
[pods field="location" helper="pods_serial_comma"]

## Examples elsewhere

n/a

## Screenshots / Screencast

n/a

**Comment by sc0ttkclark** [comment]
This would be great to add.
